web/src/pages/speedtest.rs

```diff
@@ -169,26 +171,26 @@ impl Component for Speedtest {
             }
 
             Self::Message::Start => {
-                let total_tx = self.amount;
-
-                self.state = State::Started {
-                    started_at: Local::now(),
-                    sent_tx: 0,
-                    confirmed_tx: 0,
-                    total_tx,
-                };
-
-                let users_to_send_to = self
-                    .to_username
-                    .as_ref()
-                    .and_then(|username| self.username_to_user.get(username))
-                    .map(|user| vec![user.clone()])
-                    .unwrap_or_else(|| self.username_to_user.values().cloned().collect());
-
-                self.link.send_message(Self::Message::Running {
-                    users_to_send_to,
-                    remaining: total_tx,
-                });
+                if let Some(total_tx) = validate_amount(&self.amount) {
```

Comment:
Should there be an error message if the user enters a wrong number?

Reply:
The user can't enter a wrong number using the interface, as the "Launch" button is disabled on bad input.